### PR TITLE
REST helper refactor

### PR DESCRIPTION
**Pull request by Kalostrinho**
### This PR includes:
- [X] Adjusted **_REST.js_** helper to latest practices of **ES6+** 
- [X] Implemented functionality around **_followRedirect_** to enable to user to switch on/off automatic redirects
- [X] Fix indentation

### CODE REVIEW

**Comment by Kalostrinho** [comment]
@DavertMik most are `indentation errors` from *Codacy*... Is it standard for repository code to use 2-spaces indents or 4-spaces indents? I can fix this easily, just let me know.

**Comment by DavertMik** [comment]
Just use `npm run lint-fix` to make the code clean )

File: lib/helper/REST.js [hunk 1]
```diff
@@ -41,6 +41,54 @@ class REST extends Helper {
...
+  /**
+   * Set whether 3XX request should automatically redirect
+   *
+   * ```js
+   * I.setRequestRedirects(); // To enable auto-redirects
+   * ```
+   *
+   * @param {boolean} autoRedirect
+   */
+  setRequestRedirects(autoRedirect = true) {
```
Comment (DavertMik) [inline lib/helper/REST.js]: I'm against using booleans as parameters in API as they are not so user-friendly. The `autoRedirect` won't be visible in console. Better to make functions with different names:

```
I.amFollowingRequestRedirects();
I.amNotFollowingRequestRedirects();
```

Comment (AlanAditazz) [reply lib/helper/REST.js]: Agreed! 👍

**Comment by DavertMik** [comment]
Looks like Travis builds are failing for REST tests. Can you take a look?

**Comment by Kalostrinho** [comment]
Seems that some code blocks were `padded by blank lines`
I executed `npm run lint-fix`, rebased my commit and pushed changes again, hoping this will fix the errors. Cheers!

**Comment by Kalostrinho** [comment]
This time, failures on **TravisCI**, seem to be on _Puppeteer_. _REST_ is getting through though.
```javascript
89 passing (3m)
1 failing

  1) Puppeteer
       #switchToNextTab, #switchToPreviousTab, #openNewTab, #closeCurrentTab
         should close current tab:
      expected url to include "/login"
      + expected - actual
      -about:blank
      +/login
```

**Comment by DavertMik** [comment]
yeah, no problem. If it fails only on one job that doesn't seem like a real issue. I have restarted the job

**Comment by DavertMik** [comment]
Thanks, tests passed

**Pull request merged**